tests/snappi_tests/pfcwd/test_pfcwd_actions.py

```diff
@@ -854,7 +854,7 @@ def test_pfcwd_disable_pause_cngtn(snappi_api,                  # noqa: F811
     # Percentage drop expected for lossless and lossy traffic.
     # speed_tol is speed tolerance between egress link speed and actual speed.
     # loss_expected to check losses on DUT and TGEN.
-    test_check = {'lossless': 0, 'lossy': 0, 'speed_tol': 41, 'loss_expected': False, 'pfc': True}
+    test_check = {'lossless': 0, 'lossy': 0, 'speed_tol': 55, 'loss_expected': False, 'pfc': True}
```

Comment:
@rraghav-cisco can we formularise this so that we understand in the future:


```suggestion
    test_check = {'lossless': 0, 'lossy': 0, 'speed_tol': port_map[2] * test_def['BG_FLOW_AGGR_RATE_PERCENT'], 'loss_expected': False, 'pfc': True}
```

Reply:
@auspham , I agree with you. Lets discuss bring this up in the community meeting.

Reply:
@auspham , the test_def is only defined after this line. so we can't use it here.